acinclude.m4 (AC_LBL_ENABLE_CHECK): Adapt for --enable-option-checking
introduced in 2.62.

--- bro-20080804/acinclude.m4.orig	2006-04-23 07:55:10.000000000 +0200
+++ bro-20080804/acinclude.m4	2008-08-26 16:19:45.000000000 +0200
@@ -190,9 +190,7 @@
 	sed -n -e 's/^enable_\([[^=]]*\)=[[^=]]*$/\1/p' |
 	while read var; do
 	    ok=0
-        dnl change '-' to '_' 
-        simplified=`echo $1| sed -e 's/-/_/g'`
-        for o in $simplified; do
+            for o in option_checking m4_translit([$1], -, _); do
 		    if test "${o}" = "${var}" ; then
 			    ok=1
 			    break
